python/examples/flow_example.py

```diff
@@ -0,0 +1,19 @@
+import opm.io
+from opm.io.parser import Parser, ParseContext
+from opm.io.ecl_state import EclipseState
+from opm.io.schedule import Schedule
+from opm.io.summary import SummaryConfig
+from simulators import simulators
+
+parse_context = ParseContext( [("PARSE_RANDOM_SLASH", opm.io.action.ignore)] )
+deck = Parser().parse( '../../../opm-data/norne/NORNE_ATW2013.DATA' , parse_context)
+state = EclipseState(deck)
+schedule = Schedule(deck, state)
+summary_config = SummaryConfig(deck, state, schedule)
+
+p = simulators.BlackOilSimulator()
+p.setDeck(deck)
```

Comment:
This should be simplified, either the filename should be a constructor argument:
```
p = simulators.BlackOilSimulator( "filename.DATA")
p.run( )
```
or alternatively *one* method:
```
p = simulators.BlackOilSimulator()
p.load_deck( "file.DATA")
p.run( )
```